test/e2e/egress_services.go

```diff
@@ -0,0 +1,452 @@
+package e2e
```

Comment:
@oribon have you considered what happens when egress-services is configured in a cluster _together_ with egress IPs? I would suspect OVN rules to look a bit similar in that case. Making it predictable as possible would be a big win.

It may be useful to have that integration tested to ensure it behaves as we expect it to. wdyt?

Reply:
yep, in the enhancement we decided this will use lrps with a higher priority than the eip ones - it's a good idea to have it tested so I'll write something :)

Reply:
added `Should validate egress service has higher priority than EgressIP` :)